backend/tournesol/models/entity.py

```diff
@@ -86,29 +132,34 @@ class Video(models.Model, WithFeatures, WithEmbedding):
         auto_now_add=True,
         help_text="Last time fetch of metadata was attempted",
     )
+    # TODO: specific, will be moved to the metadata
     wrong_url = models.BooleanField(default=False, help_text="Is the URL incorrect")
+    # TODO: specific, will be moved to the metadata
     is_unlisted = models.BooleanField(default=False, help_text="Is the video unlisted")
-
+    # TODO: specific, will be moved to the metadata
     tags = models.ManyToManyField(Tag, blank=True)
 
-    # computed in the Video.recompute_pareto(),
+    # computed in the Entity.recompute_pareto(),
     #  called via the manage.py compute_quantile_pareto command
     # should be computed after every ml_train command (see the devops script)
     # SEE also: {feature}_quantile fields (defined below)
 
+    # XXX is that used?
```

Comment:
Not anymore, but could be again part of a ML feature for Entities. Let's delete for now.

Reply:
Deleted.